app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheet.kt

```diff
@@ -118,6 +119,9 @@ class BloodPressureEntrySheet : BottomSheetActivity(), BloodPressureEntryUi, Rem
   @Inject
   lateinit var patientRepository: PatientRepository
 
+  @Inject
+  lateinit var uuidGenerator: UuidGenerator
```

Comment:
Should we consider moving the `BloodPressureEntryEffectHandler` to use `AssistedInject` instead of using the factory so that we can inject the `UuidGenerator` directly to effect handler instead of injecting it into `BloodPressureEntrySheet`?

Reply:
Yes, good point. I will do this.

Reply:
Done.